libnmstate/nm/active_connection.py

```diff
@@ -96,12 +101,20 @@ def _deactivate_connection_callback(self, src_object, result, user_data):
                     e,
                 )
             else:
-                self._mainloop.quit(
-                    'Connection deactivation failed on {}: error={}'.format(
-                        self._nmdev.get_iface(), e
+                if (
```

Comment:
Don't you need to check that the error is of type `GLib.GError`? Like we do in `is_action_canceled`.